src/components/autocomplete/autocomplete.spec.js

```diff
@@ -2116,6 +2116,55 @@ describe('<md-autocomplete>', function() {
       document.body.removeChild(parent[0]);
     }));
 
+    it('should allow dropdown position to be specified', inject(function($timeout, $window) {
+      var scope = createScope();
+      var dropdownPosition = 'top';
+
+      scope.match = fakeItemMatch;
+
+      var template = '<div>' +
+          '<md-autocomplete ' +
+          'md-search-text="searchText" ' +
+          'md-items="item in match(searchText)" ' +
+          'md-item-text="item" ' +
+          'md-min-length="0" ' +
+          'md-dropdown-position="' + dropdownPosition +'"' +
```

Comment:
Can you just create a scope variable for this, so you can just update the variable later?

```html
<x md-dropdown-position="{{ position }}">
```
```js
// Then later you could just do the following
scope.$apply('position = "bottom"');
```